<!DOCTYPE HTML>

<html>
	<head>
	<meta name="generator" content="Hugo 0.31.1" />
		<title>Emily S. Bellis, Ph.D.</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="/css/main.css" />
		<link rel="stylesheet" href="/css/project.css" />
		
		<noscript><link rel="stylesheet" href="/css/noscript.css" /></noscript>
	</head>
	
			<script src="/js/jquery.min.js"></script>
	<body>

    
      <div id="wrapper">

        
          <header id="header">
            <div class="logo">
              
              <img src="/images/DNA_white.png" width="100%" style="padding: 10px 10px 10px 10px;">
            
            
            </div>
            <div class="content">
              <div class="inner">
                <h2>Emily S. Bellis, Ph.D.</h2>
                <p>Welcome! I am an Assistant Professor of Bioinformatics at Arkansas State University in the Department of Computer Science. I study complex organism interactions with abiotic and biotic environments and how these interactions change across space and over time. I take inspiration from evolutionary genomics, spatial ecology, and machine learning and am particularly interested in novel intersections of these fields to address problems relevant to food security and ecosystem health in a rapidly changing climate.</p>
		<h4>I currently have paid opportunities for undergraduate students and a Ph.D. Student (through the Molecular Biosciences or Environmental Sciences programs) or Master's Student to join the lab. Please contact Emily for more information on projects of potential mutual interest!</h4></p>      
              </div>
            </div>
            <nav>
              <ul>
                
                  <li><a href="#Research">Research</a></li>
		  
		  <li><a href="#People">People</a></li>
                
                  <li><a href="#Publications">Publications</a></li>
                
                  <li><a href="#News">News</a></li>
                
                  <li><a href="#contact">Contact</a></li>
                
              </ul>
            </nav>
          </header>

        
          <div id="main">
            
              <article id="Research">
  <h1>Research</h1>
  <p>
<figure class="image main">
    
        <img src="/images/IMG_0422.jpg" />
    
    
	  </figure>
<h3>Adaptation to Environment</h3>
	<p>Understanding the genetic basis of adaptation to environment is critical for predicting and mitigating impact of rapidly changing climates on ecosystems and human societies. A primary goal of the lab is to integrate genomic, phenotypic, and environmental data to identify genetic variation important for adaptation to biotic and abiotic environments. Previous work in marine systems supports a role for adaptation to local environments via selection on the ability to associate with distinct microbial symbionts (Bellis et al. 2018, Bellis & Denver 2017). We are continuing a special focus on adaptation to biotic environments through work on coevolutionary genomics of the parasitic witchweed <i>Striga hermonthica</i> and its cereal hosts, which include sorghum, rice, millet, and maize. <i>S. hermonthica</i> is a devastating constraint to food security across much of Africa, and we are leveraging ecological and evolutionary genomic perspectives to help inform sorghum breeding and weed management in smallholder farming systems.  We are also developing novel approaches for improved understanding and prediction of plant-environment interactions, through integration of machine learning and evolutionary genomic modeling frameworks.</br>
	<ul>
		<li><b>E.S. Bellis,</b> V.L. Deleo, E.A. Kelly, C.M. Lorts, G. Rouhan, A. Budden, G.B. Bhaskara, Z. Hu, R. Muscarella, T.E. Juenger, M.P. Timko, G.K. Morris, C.W. dePamphilis, and J.R. Lasky. (In revision). Genomics of sorghum local adaptation to a parasitic plant. <a href= "https://www.biorxiv.org/content/10.1101/633529v1"><font color="MediumAquaMarine">bioRxiv</font></a></li>
		<li><b>E.S. Bellis</b>, R.B. Edlund, H.K. Berrios, H.A. Lessios, and D.R. Denver (2018). Molecular signatures of host specificity linked to habitat specialization in <i>Exaiptasia</i> sea anemones. <i> Ecology and Evolution</i>. <b>8: </b>5413-5426. <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ece3.4058">doi: 10.1002/ece3.4058 </a><a href="https://datadryad.org/resource/doi:10.5061/dryad.g8t9240"><font color="MediumAquaMarine">data/code</font></a></li>
	</ul>
	</p> 
<h3>Conflict & Cooperation</h3>
	<p>The dynamics of interactions within and between organisms and their environment is a central component of the complexity of life, from organelles that power plant and animal cells, to beneficial microbes that help lay the foundation of coral reef ecosystems, to damaging parasites of agriculture.  Previous work in this area has focused on characterizing evolutionary dynamics of selfish genetic elements in nuclear and organellar genomes of invertebrates including sea anemones, water fleas, and nematodes. Future and ongoing projects will focus on understanding how interactions between species shape dynamics of intragenomic conflict & cooperation in weedy & domesticated plants. </br>
	<ul>
		<li><b>E.S. Bellis</b> <i>et al.</i> (in prep). Intraspecific variation in transposable element accumulation over time.</li>
		<li>W.S. Philips, A.L Coleman-Hulbert, <b>E.S. Weiss</b>, D.K. Howe, S. Ping, R.I. Wernick, S. Estes, and D.R. Denver (2015). Selfish mitochondrial DNA proliferates and diversifies in small, but not large, experimental populations of <i>Caenorhabditis briggsae. Genome Biology and Evolution </i> <b>7</b>: 2023-2037. <a href="https://academic.oup.com/gbe/article/7/7/2023/631463">doi: 10.1093/gbe/evv116</a></li>
		<li>A. Emblem, S. Okkenhaug, <b>E.S. Weiss</b>, D.R. Denver, B.O. Karlsen, T. Moum, S.D. Johansen (2014). Sea anemones possess dynamic mitogenome structures. <i>Molecular Phylogenetics and Evolution</i> <b>75:</b> 184-193. <a href="http://www.sciencedirect.com/science/article/pii/S1055790314000670?via%3Dihub">doi:10.1016/j.ympev.2014.02.016</a></li>
	</ul>	      
</p>		      
</article>

            
              <article id="Publications">
<h3>Publications</h3>
<p><b>E.S. Bellis,</b> E.A. Kelly, C.M. Lorts, H. Gao, V.L. DeLeo, G. Rouhan, A. Budden, G.B. Bhaskara, Z. Hu, R. Muscarella, M.P. Timko, B. Nebie, S.M. Runo, N.D. Chilcoat, T.E. Juenger, G.P. Morris, C.W. dePamphilis, and J.R. Lasky. (Accepted). Genomics of sorghum local adaptation to a parasitic plant. <i>PNAS</i><a href= "https://www.biorxiv.org/content/10.1101/633529v1"><font color="MediumAquaMarine">bioRxiv</font></a></p>

<p>L. Lopez, <b>E.S. Bellis</b>, E. Wafula, S. Hearne, L. Honaas, P. Ralph, N. Unachukwu, C.W. dePamphilis, and J.R. Lasky. (2019). Transcriptomics of host-specific interactions in natural populations of the parasitic plant <i>Striga hermonthica. Weed Science</i>. <b>67: </b>397-411. <a href="https://www.cambridge.org/core/journals/weed-science/article/transcriptomics-of-hostspecific-interactions-in-natural-populations-of-the-parasitic-plant-purple-witchweed-striga-hermonthica/2DA8C62CCE1034AA7EA01EE8C2727556">doi:10.1017/wsc.2019.20 </a></p>

<p><b>E.S. Bellis</b>, R.B. Edlund, H.K. Berrios, H.A. Lessios, and D.R. Denver (2018). Molecular signatures of host specificity linked to habitat specialization in <i>Exaiptasia</i> sea anemones. <i> Ecology and Evolution</i>. <b>8: </b>5413-5426. <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ece3.4058">doi: 10.1002/ece3.4058 </a><a href="https://datadryad.org/resource/doi:10.5061/dryad.g8t9240"><font color="MediumAquaMarine">data/code</font></a></p>

<p><b>E.S. Bellis</b> and D.R. Denver (2017). Natural variation in responses to acute heat and cold stress in a sea anemone model system for coral bleaching. <i> The Biological Bulletin </i><b>23:</b> 168-181. <a href="https://www.journals.uchicago.edu/doi/10.1086/694890">doi: 10.1086&#47;694890 </a><a href="https://osf.io/n66y7/"><font color="MediumAquaMarine">data/code.</font></a> <a href="https://www.journals.uchicago.edu/doi/abs/10.1086/697178"><font color="#FCDFFF">Cover article</font></a></p>

<p><b>E.S. Bellis</b>, D.K. Howe, and D.R. Denver (2016). Genome-wide polymorphism and signatures of selection in the symbiotic sea anemone <i>Aiptasia.  BMC Genomics</i> <b>17:</b> 160. <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-2488-6">doi: 10.1186/s12864-016-2488-6</a> <a href="https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA304763"><font color="MediumAquaMarine">data</font></a></p>

<p>W.S. Philips, A.L Coleman-Hulbert, <b>E.S. Weiss</b>, D.K. Howe, S. Ping, R.I. Wernick, S. Estes, and D.R. Denver (2015). Selfish mitochondrial DNA proliferates and diversifies in small, but not large, experimental populations of <i>Caenorhabditis briggsae. Genome Biology and Evolution </i> <b>7</b>: 2023-2037. <a href="https://academic.oup.com/gbe/article/7/7/2023/631463">doi: 10.1093/gbe/evv116</a></p>

<p>A. Emblem, S. Okkenhaug, <b>E.S. Weiss</b>, D.R. Denver, B.O. Karlsen, T. Moum, S.D. Johansen (2014). Sea anemones possess dynamic mitogenome structures. <i>Molecular Phylogenetics and Evolution</i> <b>75:</b> 184-193. <a href="http://www.sciencedirect.com/science/article/pii/S1055790314000670?via%3Dihub">doi:10.1016/j.ympev.2014.02.016</a></p>

<p><h5>Preprints/Submitted</h5>
<p>R.M. Gutaker, S.C. Groen, <b>E.S. Bellis,</b> J.Y. Choi, I.S. Pires, R.K. Bocinsky, E. Slayton, O. Wilkins, C.C. Castillo, S. Negrão, M.M. Oliveira, D.Q. Fuller, J.A. d’Alpoim Guedes, J.R. Lasky, and M.D. Purugganan. (Submitted). Genomic history and ecology of the geographic spread of rice. <a href="https://www.biorxiv.org/content/10.1101/748178v1"><font color="MediumAquaMarine">bioRxiv</font></a></p> 

<p>M. Staton, C. Addo-Quaye, [and 29 others including <b>E.S. Bellis</b>]. (Submitted). The Chinese chestnut genome: a reference for species restoration. <a href= "https://www.biorxiv.org/content/10.1101/615047v1"><font color="MediumAquaMarine">bioRxiv</font></a></p>

<p>W. Zhou, <b>E.S. Bellis,</b> J. Stubblefield, J. Causey, J. Qualls, K. Walker, and X. Huang. (Submitted). Minor QTLs mining through combination of GWAS and machine learning feature selection.</p>

</article>
		  
	      <article id="People">
<h3>Emily Bellis, Ph.D.</h3>
<p><IMG SRC="em-bellis.github.io/images/EB_Pascal2019.JPG" ALIGN="right" height="118" width="107"/> Emily is from Arkansas and completed a B.S. in Genetics & Biochemistry at Texas A&M University (College Station, TX) in 2010 and a Ph.D. in Integrative Biology in 2017 at Oregon State University (Corvallis, OR), working with Dee Denver. After a short postdoc on population genomics of transposable elements with Sarah Schaack at Reed College (Portland, OR), she worked with Jesse Lasky and Claude dePamphilis at Penn State (State College, PA) as an NSF Postdoctoral Research Fellow in Biology through the National Plant Genome Initiative. She joined A-State in late 2019 as a Research Assistant Professor of Bioinformatics.  In her spare time, she enjoys hiking, knitting, and attempting unsuccessfully to tire out her energetic Border Collie/Aussie, Pascal.</br>   
<a href="/images/EBellis_CV_10.2019.pdf"><font color="MediumAquaMarine">Emily's CV</font></a> </br>
<h3>You?</h3>
We're recruiting undergraduate students and a Master's/Ph.D. student to work on questions related to genomics of interactions with environment!  The lab currently focuses on domesticated and weedy plant species (e.g. sorghum, <i>Striga hermonthica</i>) as model systems. Please contact Emily to discuss particular research projects you might pursue in the lab to help you progress along your desired career path. 
		      
</article>
            
              <article id="News">
  <h1>News</h1>
  <p>
<figure class="image main">
    
        <img src="/images/IMG_0400.png" />
    
    
</figure>

<b>10.25.19</b> Emily is off to the annual Arkansas INBRE conference, in Fayetteville, AR!

</article>

            
              <article id="contact">
  <h1>Contact</h1>
<p>You can find me in room 202 of the Arkansas Biosciences Institute on the A-State main campus in Jonesboro, or reach me by email at ebellis[at]astate[dot]edu.</p>

<ul class="icons">

	<li><a href="https://twitter.com/em_bellis" class="icon fa-twitter"><span class="label">twitter</span></a></li>

	<li><a href="https://github.com/em-bellis" class="icon fa-github"><span class="label">github</span></a></li>

</ul>

</article>

            
          </div>

        
          <div id="languages">
            
          </div>

				
					<footer id="footer">
						<p class="copyright">&copy; Untitled. Design: <a href="https://html5up.net">HTML5 UP</a>. Powered by <a href="https://www.gohugo.io/">Hugo.</a></p>
					</footer>

			</div>
		
			<div id="bg"></div>

		
			<script src="/js/skel.min.js"></script>
			<script src="/js/util.js"></script>
			<script src="/js/main.js"></script>
			

	</body>
</html>

