<!DOCTYPE HTML>

<html>
	<head>
	<meta name="generator" content="Hugo 0.31.1" /> 
		<title>Eco-Evolutionary Informatics Lab @A-State</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="/css/main.css" />
		<link rel="stylesheet" href="/css/project.css" />
		
		<noscript><link rel="stylesheet" href="/css/noscript.css" /></noscript>
	</head>
	
			<script src="/js/jquery.min.js"></script>
	<body>

    
      <div id="wrapper">

        
          <header id="header">
            <div class="logo">
              
              <img src="/images/DNA_white.png" width="100%" style="padding: 10px 10px 10px 10px;">
            
            
            </div>
            <div class="content">
              <div class="inner">
                <p>Welcome to the Eco-Evo Informatics Lab at Arkanasas State University! We use primarily computational approaches to study (mostly plant) interactions with abiotic and biotic environments and how these interactions change across space and over time. As part of the <a href="https://www.astate.edu/a/cnbt/index.dot">Center for No-Boundary Thinking</a> we attempt to define our major research questions without the boundary of disciplines and are passionate about promoting deep feedback between machine learning and other fields, including evolutionary genomics, spatial ecology, and plant science.</p>
		 <h4> As of May 2023, the EcoEvoInfo Lab is closing its doors as Dr. Bellis is taking a new position as Computational Biology Lead at <a href = "https://www.avalo.ai">Avalo</a>. See how our A-State collaborations are continuing by checking out our current <a href="#Projects">projects</a> and whats next for each member of our <a href="#Team">team.</a></h4>   
		
              </div>
            </div>
            <nav>
              <ul>
                
                  <li><a href="#Projects">Projects</a></li>
		      
		  <li><a href="#Team">Team</a></li>
                
                  <li><a href="#News">News</a></li>
                
                  <li><a href="#contact">Contact</a></li>
                
              </ul>
            </nav>
          </header>

        
          <div id="main">
            
              <article id="Projects">
  <h1>Current Projects</h1>
  <p>
<figure class="image main">
    
        <img src="/images/IMG_0422.jpg" />
    
    
	  </figure>
<h3> G &times E</h3>
<p> We are interested in understanding how genotype-by-environment interaction (when response to environmental variation depends on an organism's genotype) influences variation in phenotype. Currently we are working on approaches to better integrate multimodal datasets (genomic, environmental, and phenotypic) in machine learning frameworks to predict fitness and trait variation across different environments, and to identify the underlying genetic mechanisms of this variation. We are particularly interested in data structures with complexity in multiple domains (such as multitemporal, multispectral images from unmanned aerial vehicles or evolutionary genomic data from georeferened crop landraces). This research is currently supported by an <a href="https://www.arkansasedc.com/science-technology/division/data-analytics-that-are-robust-trusted-(dart)">Arkansas NSF EPSCoR DART seed grant</a> and through the University of Arkansas Division of Agriculture.</br></p> 
<h4>Related Publications:</h4>
	<ul>
		<ul>
			<li><font color="MediumAquaMarine">Bellis, Hashem, et al. (2022) <a href="https://www.frontiersin.org/articles/10.3389/fpls.2022.716506/full"><i>Frontiers in Plant Science</i></a>.</font> Our first foray into computer vision models for detecting plant stress!</li>
		</ul>
	</ul>	 

<h3> Biotic Environments</h3>
<p> Much of our past and ongoing work seeks to understand spatial dynamics of interactions between hosts and their parasite or mutualist partners. We have worked in a broad range of systems, from marine invertebrates and their symbiotic microbes to parasitic plants (<i>Striga hermonthica</i>) and their cereal hosts (including sorghum, rice, millet, and maize).</p> 
<p> Current projects are leveraging ecological & evolutionary genomic perspectives to understand how parasitic plants and their hosts coevolve across broad spatial scales and diverse environmental gradients. This work is funded through an <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=2151820&HistoricalAwards=false"><font color="MediumAquaMarine">NSF National Research Traineeship.</font></a></br></p>
<h4>Related Publications:</h4>
	<ul>
		<ul>
			<li><font color="MediumAquaMarine">Bellis et al. (2020) <a href="https://www.pnas.org/doi/epdf/10.1073/pnas.1908707117"><i>PNAS</i></a></font>. We developed a new framework to identify host alleles associated with adaptation to parasites across the range of the staple crop, sorghum.</li>
			<li><font color="MediumAquaMarine">Bellis, McLaughlin, et al. (2021) <a href="https://onlinelibrary.wiley.com/doi/10.1111/ecog.05730"><i>Ecography</i></a></font>. In this study, we find that patterns of parasite adaptation to host communities emerge at continental scale and can be reasonably well predicted from variation in parasite abiotic environmental niche.</li>
			<li><font color="MediumAquaMarine">Bellis et al. (2022) <a href="https://www.biorxiv.org/content/10.1101/2022.02.01.478712v1"><i>bioRxiv</i></a></font>. Here we uncover genomic signatures of parasite adaptation to different hosts, using a reference-free approach.</li>
					</ul>
	</ul>

<h3> Novel Environments</h3>
<p> Invasive species are fascinating model systems for studying how organisms can persist and rapidly adapt to new environmental conditions. We are contributing to a variety of projects that seek to predict how non-native invasive plants will perform in novel environments and to understand the factors shaping their success. In particular, we are interested in how plants with complex genomes adapt to novel environments, with a focus on wild sugarcane, a Federal Noxious Weed that so far is known to be established in the continental US only in Florida (Funding: USFS-Southern Research Station). </br></p>  
<h4>Related Publications:</h4>
	<ul>
		<ul>
			<li><font color="MediumAquaMarine">Lucardi et al. (2020) <a href="https://www.nature.com/articles/s41598-020-71954-3"><i>Scientific Reports</i></a>.</font> This is the study that launched our work on genomics of wild sugarcane. We estimated that tens of thousands of propagules are entering the U.S. each year through global trade routes.</li>
			<li><font color="MediumAquaMarine">Bellis et al. (2022) <a href="https://bsapubs.onlinelibrary.wiley.com/doi/10.1002/ajb2.16079"><i>American Journal of Botany</i></a>.</font> Our invited essay on levaraging genomics to predict adaptive potential of invasive grasses. </li>
		</ul>
	</ul>

</article>
		  
	      <article id="Team">	      
<p>
<figure class="image main">
    
        <img src="/images/montgomery2018.jpg" />
    
    
	  </figure>
	
<p>Our group welcomes individuals from diverse backgrounds and strives to support all team members in an vibrant, respectful, and supportive atmosphere. The organization of our group is inspired by the 'nested multi-mentoring model' described by Montgomery & Page <a href = "https://www.nap.edu/resource/25568/Montgomery%20and%20Page%20-%20Mentoring.pdf">(2018)</a>. In this type of mentoring model, multiple mentors work collaboratively to support mentees, who also support each other and benefit from multiple mentors and resources.</p>

<p>Members of the Bellis 'subnetwork' are encouraged to attend a weekly lab group meeting with several other faculty members in the Bioinformatics working group, as well as to engage with other students and faculty members across the country as part of the <a href="https://aicamp.us">AI-Campus</a> program. </p>

<h2>Current Team </h2>
<h3>Emily Bellis, Ph.D. (PI) </h3>
<p><IMG SRC="/images/Emily_photo.jpg" ALIGN="left" height="118" width="107"/> Dr. Bellis earned a B.S. in Genetics and Biochemistry at Texas A&M University in 2010 and a Ph.D. in Integrative Biology in 2017 from Oregon State University. She began working in plant systems as an NSF Postdoctoral Research Fellow at Penn State, where her research focused on spatial coevolutionary genomics of cereal-parasitic plant interactions. In 2019, she returned to her home state of Arkansas where she was an Assistant Professor of Bioinformatics at Arkansas State University until May 2023. At A-State, she also served as Associate Director of the Center for No-Boundary Thinking leading several initiatives in interdisciplinary data science research and education including for an NSF Research Traineeship Program UandI-DEECoDE. In May 2023, she will take on a new role as Computational Biology Lead at Avalo, a start-up based in Durham, NC, where she will continue research to promote feedback between machine learning, evolutionary genomics, spatial ecology, and plant science, towards more resilient and sustainable food systems. She is continuing several collaborations with A-State researchers and service on several thesis advisory committees. <a href="https://github.com/em-bellis/CV/raw/master/Bellis_CV.pdf"><font color="MediumAquaMarine">Emily's CV.</font></a></br>   

<h3>Natalie Haydt, M.S. (Ph.D. Student) </h3>
<p> <IMG SRC="/images/natalie.jpg" ALIGN="left" height="118" width="107"/> Natalie worked in the EcoEvoInfo lab for two years as a Ph.D. student in the Molecular Biosciences Program, co-advised with Dr. Lori Neuman-Lee. Her research integrates modeling, high-throughput sequencing, and experiments to understand disease dynamics and spread in wild vertebrates (snakes) and an emerging fungal pathogen. She is currently continuing her research in Dr. Neuman-Lee's lab. More about Natalie's research <a href = "https://nhaydt.owlstown.net"><font color="MediumAquaMarine">here.</font></a>  

<h3>Fared Farag (M.S. Student) </h3>
<p> <IMG SRC="/images/shelby.png" ALIGN="left" height="118" width="107"/> Fared is beginning his second year as an M.S. student in Computer Science (emphasis in Data Science), co-advised with Dr. Ahmed Hashem and Dr. Jason Causey. His thesis is on developing a nitrogen recommender system to utilize remote sensing in detecting nitrogen deficiencies in rice. On the side, he is also interested in Backend Web Development. <a href="/images/Fared-resume.pdf"><font color="MediumAquaMarine">Fared's resume.</font></a>  	

<h3>Indy Hughes (M.S. Student) </h3>
<p> <IMG SRC="/images/indy.jpg" ALIGN="left" height="118" width="107"/> Indy is beginning her second year as an M.S. student in Biology. She is leveraging genomic analyses to understand the origins and adaptive potential of naturalizing wild sugarcane populations and their hybrids in the United States. She is continuing her thesis research under primary advisement of Dr. Travis Marsico. </a>  
	
<h3>Emilio Soriano Chavez (Undergraduate Researcher) </h3>
<p> <IMG SRC="/images/Emilio0SC.JPG" ALIGN="left" height="118" width="107"/> Emilio is a senior at A-State majoring in Computer Science. He is currently working on a project to develop new computational models to predict phenotype from environmental and genotype information in maize. He is currently studying abroad for the summer but will resume research in the area of cybersecurity with Dr. Donghoon Kim when he returns to A-State in Fall 2023.</br>

<h3>Shelby McCormick (Undergraduate Researcher) </h3>
<p> <IMG SRC="/images/shelby2.jpg" ALIGN="left" height="118" width="107"/> Shelby is a senior Mathematics major. Her project focuses on training deep learning models for automated detection of phenological stage from herbarium specimen images of <i>Striga</i> spp. parasitic plants.</br></br>

<h3>Yixing Wang (Undergraduate Researcher) </h3>
<p> <IMG SRC="/images/yixing.jpg" ALIGN="left" height="118" width="107"/> Yixing is a senior at A-State majoring in Computer Science, studying biologically-inspired algorithms. His current project is using knowledge of nervous system evolution to inform the design of neural network architectures. Yixing is now pursuing his research interests in a computational neuroscience in a summer internship at University of Arkansas for Medical Sciences in Little Rock.</br>

<h3>Current opportunities:</h3>	

<p>The EcoEvoInfo Lab is no longer recruiting undergraduate, M.S., or Ph.D. students. However, there are other opportunities for related research at A-State as part of a new NSF-funded National Research Traineeship program, UandI-DEECoDE. Check out the press release <a href="https://beta.nsf.gov/news/nsf-research-traineeship-program-expands-include-43-states"><font color="MediumAquaMarine">here</a></font> and the <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=2151820&HistoricalAwards=false"><font color="MediumAquaMarine">award abstract</font></a> for more information about the program, or contact <a href="https://www.astate.edu/uandideecode">Greg Umhoefer</a> for more information!</p>

<p>Additionally, we will be recruiting a postdoctoral researcher as part of a collaboration between Avalo and the Lorence lab at A-State to help lead a project focused on using machine learning to identify genetic variants associated with adaptation of rice to low input environments. The position includes a 3-month industry internship at Avalo. Contact Dr. Argelia Lorence or Dr. Bellis for details. </p>


</article>
            
              <article id="News">
  <h1>News</h1>
  <p>
<b>10.04.22</b> Emily was recognized as the 2022 ABI New Investigator of the Year! Check out the press release <a href="https://aaes.uada.edu/news/emily-bellis-abi-researcher/">here.</a> </p>
<b>9.15.22</b> Our grant to start a new NSF National Research Traineeship at A-State was recommended for funding! Listen to our interview on KASU <a href="https://www.kasu.org/show/a-state-connections/2022-09-15/decode-project-garners-2-million-grant-from-national-science-foundation">here. </a> </p>
<b>6.15.22</b> Welcome to new lab members Indy, Yixing, and Fared! Indy & Fared are starting as M.S. students in Biology and Computer Science, respectively and Yixing is an ABI Summer Undergraduate Scholar! Shelby was awarded a DART SURE Scholarship to work in the lab over the summer. Emily was featured as a new faculty member in the 20th Anniversivery ABI Annual Report (<a href="https://arbiosciences.org">p. 34!</a>). Congrats all! </p>
	  
<b>4.22.22</b> Very proud of Natalie, Shelby, and Emilio this week for their awesome poster presentations at Create@State! Natalie also won the Dean's Award for best graduate poster presentation for the College of Math & Science! </p>
	  
<b>4.03.22</b> Congrats Shelby and Emilio, who both presented their research results at the Clinton Presidential Library in Little Rock, as part of the NSF EPSCoR Artificial Intelligence with No-Boundary Thinking workshop!</p>
		      <p> <IMG SRC="/images/emilio_lr.jpg" ALIGN="right" height="240" width="320"/><IMG SRC="/images/shelby_lr.jpg" ALIGN="center" height="320" width="240"/></p>  
	 
<b>11.22.21</b> Shelby's application for the Undergraduate Research Scholar was accepted. Woohooo!! Building on her work for the AI-Campus project, her research will focus on developing improved machine learning models for herbarium image analysis!</p>

<b>4.23.21</b> Undergraduate student researchers Daniel Delgadillo and Shelby McCormick are joining our AI-Campus team to work on the herbarium image project!  Natalie Haydt is also joining the lab to pursue her PhD in MBS, co-advised with Dr. Neuman-Lee. We are so excited to welcome Daniel, Shelby, and Natalie! </p>
	
<b>4.21.21</b> <IMG SRC="/images/prairie.JPG" ALIGN="right" height="120" width="160"/>An article <a href='https://www.arkansasonline.com/news/2021/apr/18/tallgrass-prairie-restoration-a-priority-for-fort/'>just published</a> in the NW Arkansas Democrat Gazette about the Tallgrass Prairie Restoration Project. The article came out just as we were all together sampling hemiparasitic plant associated soils in NW AR! Here's Dr. Bellis perfecting her soil coring technique under the tutelage of collaborators Mr. Jay Randolph & Dr. Jeff Shaver! </p>
	  
<b>6.09.20</b> The lab is growing! Undergraduate student Emilio Soriano-Chavez joined us this summer to work on machine learning models for genomic prediction! </p>
	  
<b>10.25.19</b> Emily is off to the annual Arkansas INBRE conference, in Fayetteville, AR!

</article>

            
              <article id="contact">
  <h1>Contact</h1>
<p>You can find me in room 202 of the Arkansas Biosciences Institute on the A-State main campus in Jonesboro, or reach me by email at ebellis[at]astate[dot]edu.</p>

<ul class="icons">

	<li><a href="https://twitter.com/em_bellis" class="icon fa-twitter"><span class="label">twitter</span></a></li>

	<li><a href="https://github.com/em-bellis" class="icon fa-github"><span class="label">github</span></a></li>

</ul>

</article>

            
          </div>

        
          <div id="languages">
            
          </div>

				
					<footer id="footer">
						<p class="copyright">&copy; Untitled. Design: <a href="https://html5up.net">HTML5 UP</a>. Powered by <a href="https://www.gohugo.io/">Hugo.</a></p>
					</footer>

			</div>
		
			<div id="bg"></div>

		
			<script src="/js/skel.min.js"></script>
			<script src="/js/util.js"></script>
			<script src="/js/main.js"></script>
			

	</body>
</html>
