<!DOCTYPE HTML>

<html>
	<head>
	<meta name="generator" content="Hugo 0.31.1" /> 
		<title>Emily S. Bellis, Ph.D.</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="/css/main.css" />
		<link rel="stylesheet" href="/css/project.css" />
		
		<noscript><link rel="stylesheet" href="/css/noscript.css" /></noscript>
	</head>
	
			<script src="/js/jquery.min.js"></script>
	<body>

    
      <div id="wrapper">

        
          <header id="header">
            <div class="logo">
              
              <img src="/images/DNA_white.png" width="100%" style="padding: 10px 10px 10px 10px;">
            
            
            </div>
            <div class="content">
              <div class="inner">
                <p>Welcome to the Bellis sub-network at Arkanasas State University! We study organism interactions with abiotic and biotic environments and how these interactions change across space and over time. As part of the <a href="https://cnbt-astate.com">Center for No-Boundary Thinking</a> we attempt to define our major research questions without the boundary of disciplines (though we frequently incorporate perspectives from evolutionary genomics, machine learning, spatial ecology, and data science research communities).</p>
		      <p><h5>spatial coevolutionary genomics; G X E; parasitic plants; machine learning; interdisciplinarity </h5></p>
		
              </div>
            </div>
            <nav>
              <ul>
                
                  <li><a href="#Projects">Projects</a></li>
		  
		  <li><a href="#Team">Team</a></li>
                
                  <li><a href="#Publications">Publications</a></li>
                
                  <li><a href="#News">News</a></li>
                
                  <li><a href="#contact">Contact</a></li>
                
              </ul>
            </nav>
          </header>

        
          <div id="main">
            
              <article id="Projects">
  <h1>Current Projects</h1>
  <p>
<figure class="image main">
    
        <img src="/images/IMG_0422.jpg" />
    
    
	  </figure>
	 

<h3> Spatial Coevolutionary Genomics</h3>
<p> Much of our past and ongoing work seeks to understand spatial dynamics of interactions between hosts and their parasite or mutualist partners. We have worked in a broad range of systems, from marine invertebrates and their symbiotic microbes to parasitic plants (<i>Striga hermonthica</i>) and their cereal hosts (including sorghum, rice, millet, and maize).</p> 
<p> Current projects are leveraging ecological & evolutionary genomic perspectives to understand how parasites adapt across broad spatial scales and diverse environmental gradients. </br></p>
<h4>Related Publications:</h4>
	<ul>
		<ul>
			<li>Bellis et al. (2022) <a href="https://www.biorxiv.org/content/10.1101/2022.02.01.478712v1"><i>bioRxiv</i></a>.</li>
			<li>Bellis, McLaughlin, et al. (2021) <a href="https://onlinelibrary.wiley.com/doi/10.1111/ecog.05730"><i>Ecography</i></a>.</li>
			<li>Bellis et al. (2020) <a href="https://www.pnas.org/doi/epdf/10.1073/pnas.1908707117"><i>PNAS</i></a>.</li>
		</ul>
	</ul>
		
<h3> Phenotype Prediction across Environments</h3>
<p> We are interested in understanding how genotype-by-environment interaction (when response to environmental variation depends on an organism's genotype) influences variation in phenotype. Currently we are working on approaches to better integrate datasets incorporating multiple data types (genomic, environmental, phenotypic, and image-based data) in machine learning frameworks to predict fitness and trait variation across different environments, leveraging the extensive data and germplasm resources available for crop systems. We are also working on development of methods for plant phenotyping using multispectral images from unmanned aerial vehicles (UAVs). This research is supported by an <a href="https://www.arkansasedc.com/science-technology/division/data-analytics-that-are-robust-trusted-(dart)">Arkansas NSF EPSCoR DART seed grant</a> and through the University of Arkansas Division of Agriculture.</br></p> 
<h4>Related Publications:</h4>
	<ul>
		<ul>
			<li>Bellis, Hashem, et al. (2022) <a href="https://www.frontiersin.org/articles/10.3389/fpls.2022.716506/full"><i>Frontiers in Plant Science</i></a>.</li>
		</ul>
	</ul>

<h3> Adaptation to Novel Environments</h3>
<p> Invasive species are fascinating model systems for studying how organisms can persist and rapidly adapt to new environmental conditions. We are contributing to a variety of projects that seek to predict how non-native invasive plants will perform in novel environments and understanding the factors shaping their success (Funding: USFS-Southern Research Station). </br></p>  
<h4>Related Publications:</h4>
	<ul>
		<ul>
			<li>Masanga et al. (2021) <a href="<a href="https://academic.oup.com/plphys/advance-article/doi/10.1093/plphys/kiab034/6123756"><i>Plant Physiology</i></a>.</li>
			<li>Lucardi et al. (2020) <a href="<a href="https://www.nature.com/articles/s41598-020-71954-3"><i>Scientific Reports</i></a>.</li>
		</ul>
	</ul>

</article>

            
              <article id="Publications">
<h3>Publications (lab members/mentees in bold)</h3>
<p><h5>Peer-reviewed</h5>
<p> <b>E.S. Bellis</b>*, A.A. Hashem*, J.L. Causey, B.R.K. Runkle, B. Moreno-García, B. Burns, V.S. Green, T.N. Burcham, M.L. Reba and X. Huang. (Accepted). Detecting intra-field variation in rice yield with UAV imagery and deep learning. <i>Frontiers in Plant Science.</i> *co-first authors.</p> 
		      
<p> E.K.H. Ho*, <b>E.S. Bellis</b>*, J. Calkins, J.R. Adrion, L.C. Latta IV, & S. Schaack. (2021) <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009827">Engines of change: Transposable element mutation rates are high and variable within <i>Daphnia magna</i>.</a> <i>PLoS Genetics </i><b>17:</b> e1009827. *co-first authors.</p>		      		      

<p>J. Masanga, R. Odour, A. Alakonya, P. Ojola, <b>E.S. Bellis</b>*, and S.M. Runo*. (2021). <a href="https://nph.onlinelibrary.wiley.com/doi/10.1002/ppp3.10236">Comparative phylogeographic analysis of <i>Cuscuta campestris</i> and <i>C. reflexa</i> in Kenya: Implications for management of highly invasive vines.</a><i> Plants, People, Planet</i><b>4: </b> 182-193. <a href="https://nph.onlinelibrary.wiley.com/doi/10.1002/ppp3.10255"><font color="#FCDFFF">cover article</font></a> *co-corresponding.</p>

<p><b>E.S. Bellis</b>*, C.M. McLaughlin*, C.W. dePamphilis, & J.R. Lasky. <a href="https://onlinelibrary.wiley.com/doi/10.1111/ecog.05730">The geography of parasite local adaptation to host communities</a> (2021). <i>Ecography</i> 44: 1205-1217. *co-first authors. </p>

<p>J. Masanga, B. N. Mwangi, W. Kibet, M. Wamalwa, P. Sagero, R. Oduor, M. Ngugi, A. Alakonya, P. Ojola, <b>E. Bellis</b>, S. Runo. (2021). <a href="https://academic.oup.com/plphys/advance-article/doi/10.1093/plphys/kiab034/6123756">Physiological and ecological warnings that Dodder pose an exigent threat to farmlands in Eastern Africa. </a><i> Plant Physiology</i><b>4</b>: 1457-1467.</p>

<p><b>E.S. Bellis,</b> E.A. Kelly, C.M. Lorts, H. Gao, V.L. DeLeo, G. Rouhan, A. Budden, G.B. Bhaskara, Z. Hu, R. Muscarella, M.P. Timko, B. Nebie, S.M. Runo, N.D. Chilcoat, T.E. Juenger, G.P. Morris, C.W. dePamphilis, and J.R. Lasky. (2020). <a href="http://dx.doi.org/10.1073/pnas.1908707117">Genomics of sorghum local adaptation to a parasitic plant.</a> <i>PNAS </i><b>117</b>: 4243-4251. <a href= "https://science.psu.edu/news/Lasky2-2020"><font color="#FCDFFF">press</font></a></p>

<p>R.M. Gutaker, S.C. Groen, <b>E.S. Bellis,</b> J.Y. Choi, I.S. Pires, R.K. Bocinsky, E. Slayton, O. Wilkins, C.C. Castillo, S. Negrão, M.M. Oliveira, D.Q. Fuller, J.A. d’Alpoim Guedes, J.R. Lasky, and M.D. Purugganan. (2020). <a href="https://www.nature.com/articles/s41477-020-0659-6">Genomic history and ecology of the geographic spread of rice.</a><i> Nature Plants </i><b>6:</b> 492-502. <a href="https://github.com/em-bellis/riceTravelTime"><font color="MediumAquaMarine">code</font></a></p> 

<p>J. Stubblefield, M. Hervert, J.L. Causey, J.A. Qualls, W. Dong, L. Cai, J. Fowler, <b>E. Bellis</b>, K. Walker, J.H. Moore, S. Nehring & X. Huang. (2020). <a href="https://www.nature.com/articles/s41598-020-78060-4">Transfer learning with chest X-rays for ER patient classification. </a><i>Scientific Reports </i>10: 20900.</p>
	
<p>R.D. Lucardi, <b>E.S. Bellis</b>, C.E. Cunard, J.K. Gravesande, S.C. Hughes, L.E. Whitehurst, S.J. Worthy, K.S. Burgess, T.D. Marsico. (2020). <a href="https://www.nature.com/articles/s41598-020-71954-3">Seeds attached to refrigerated shipping containers represent a substantial risk of nonnative plant species introduction and establishment. </a><i>Scientific Reports </i>10: 15017.</p> 		      
		      
<p>L. Lopez, K. Turner, <b>E.S. Bellis</b>, & J.R. Lasky. (2020). <a href="https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.13245">Genomics of Natural History Collections for Understanding Evolution in the Wild.</a> <i>Molecular Ecology Resources</i> 20: 1153-1160.</p>

<p>J.S. Shaver, <b>E.S. Bellis</b>, C. Iwaki, J. Qualls, J. Randolph, & J. Smith. (2020). Massard Prairie Restoration and Soil Microbiome Succession. <i>Journal of the Arkansas Academy of Science </i>74. <font color="#FCDFFF">cover article</font></p> 
		      
<p>M. Staton, C. Addo-Quaye, [and 29 others including <b>E.S. Bellis</b>]. (2020). <a href="https://link.springer.com/article/10.1007/s11295-020-01454-y">A reference genome assembly and adaptive trait analysis of <i>Castanea mollissima</i> 'Vanuxem', a source of resistance to chestnut blight in restoration breeding. </a><i>Tree Genetics & Genomes </i>16: 57. <a href= "https://www.biorxiv.org/content/10.1101/615047v1"><font color="MediumAquaMarine">bioRxiv</font></a></p>
		      
<p>L. Lopez, <b>E.S. Bellis</b>, E. Wafula, S. Hearne, L. Honaas, P. Ralph, N. Unachukwu, C.W. dePamphilis, and J.R. Lasky. (2019). <a href="https://www.cambridge.org/core/journals/weed-science/article/transcriptomics-of-hostspecific-interactions-in-natural-populations-of-the-parasitic-plant-purple-witchweed-striga-hermonthica/2DA8C62CCE1034AA7EA01EE8C2727556">Transcriptomics of host-specific interactions in natural populations of the parasitic plant <i>Striga hermonthica.</a> Weed Science</i> <b>67: </b>397-411. </p>

<p><b>E.S. Bellis, R.B. Edlund</b>, H.K. Berrios, H.A. Lessios, and D.R. Denver (2018). <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ece3.4058">Molecular signatures of host specificity linked to habitat specialization in <i>Exaiptasia</i> sea anemones.</a><i> Ecology and Evolution</i> <b>8: </b>5413-5426. <a href="https://datadryad.org/resource/doi:10.5061/dryad.g8t9240"><font color="MediumAquaMarine">data/code</font></a></p>

<p><b>E.S. Bellis</b> and D.R. Denver (2017). <a href="https://www.journals.uchicago.edu/doi/10.1086/694890">Natural variation in responses to acute heat and cold stress in a sea anemone model system for coral bleaching.</a> <i> The Biological Bulletin </i><b>23:</b> 168-181. <a href="https://osf.io/n66y7/"><font color="MediumAquaMarine">data/code.</font></a> <a href="https://www.journals.uchicago.edu/doi/abs/10.1086/697178"><font color="#FCDFFF">cover article</font></a></p>

<p><b>E.S. Bellis</b>, D.K. Howe, and D.R. Denver (2016). <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-2488-6">Genome-wide polymorphism and signatures of selection in the symbiotic sea anemone <i>Aiptasia.</a>  BMC Genomics</i> <b>17:</b> 160. <a href="https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA304763"><font color="MediumAquaMarine">data</font></a></p>

<p>W.S. Philips, A.L Coleman-Hulbert, <b>E.S. Weiss</b>, D.K. Howe, S. Ping, R.I. Wernick, S. Estes, and D.R. Denver (2015). <a href="https://academic.oup.com/gbe/article/7/7/2023/631463">Selfish mitochondrial DNA proliferates and diversifies in small, but not large, experimental populations of <i>Caenorhabditis briggsae</a>. Genome Biology and Evolution </i> <b>7</b>: 2023-2037.</p>

<p>A. Emblem, S. Okkenhaug, <b>E.S. Weiss</b>, D.R. Denver, B.O. Karlsen, T. Moum, S.D. Johansen (2014). <a href="http://www.sciencedirect.com/science/article/pii/S1055790314000670?via%3Dihub">Sea anemones possess dynamic mitogenome structures. </a> <i>Molecular Phylogenetics and Evolution</i> <b>75:</b> 184-193. </p>
	
<p><h5>Preprints/Submitted</h5>

<p><b>C. Yim, E.S. Bellis</b>, V.L. DeLeo, D. Gamba, R. Muscarella, and J.R. Lasky (2022). <a href="https://www.biorxiv.org/content/10.1101/2022.03.06.483202v1.full">Climate biogeography of Arabidopsis thaliana: linking distribution models, individual performance, and life history.</a> bioRxiv.</p>
		      
<p><b>E.S. Bellis</b>, C.S. von Münchow, C.O. Odero, <b>A. Kronberger</b>, E. Kelly, <b>T. Xia</b>, X. Huang, S. Wicke, S.M. Runo, C.W. dePamphilis, J.R. Lasky. <a href="https://www.biorxiv.org/content/10.1101/633529v1.full">Genomic signatures of host-specific selection in a parasitic plant.</a> bioRxiv. </p> 

<p> W. Zhou, <b>E. Bellis</b>, J. Stubblefield, J. L. Causey, Jake A. Qualls, Karl Walker, Xiuzhen Huang (2019). Minor QTLs mining through the combination of GWAS and machine learning feature selection. biorxiv.  Final Version. </p>

		    
		      
</article>
		  
	      <article id="Team">	      
<p>
<figure class="image main">
    
        <img src="/images/montgomery2018.jpg" />
    
    
	  </figure>
	
<p>Our group welcomes individuals from diverse backgrounds and strives to support all team members in an vibrant, respectful, and supportive atmosphere. The organization of our group is inspired by the 'nested multi-mentoring model' described by Montgomery & Page <a href = "https://www.nap.edu/resource/25568/Montgomery%20and%20Page%20-%20Mentoring.pdf">(2018)</a>. In this type of mentoring model, multiple mentors work collaboratively to support mentees, who also support each other and benefit from multiple mentors and resources.</p>

<p>Members of the Bellis 'subnetwork' are encouraged to attend a weekly lab group meeting with several other faculty members in the Bioinformatics working group, as well as to engage with other students and faculty members across the country as part of the <a href="https://aicamp.us">AI-Campus</a> program. </p>

<h2>Current Team </h2>
<h3>Emily Bellis, Ph.D. (PI) </h3>
<p><IMG SRC="/images/Emily_photo.jpg" ALIGN="left" height="118" width="107"/> Emily is from Arkansas and earned a B.S. in Genetics & Biochemistry at Texas A&M University in 2010 and a Ph.D. in 2017 from Oregon State University where her dissertation research focused on coevolutionary genomics. She joined A-State in late 2019 after a postdoc at Penn State and is currently an Assistant Professor of Bioinformatics.</br>   

<h3>Natalie Haydt, M.S. (Ph.D. Student) </h3>
<p> <IMG SRC="/images/natalie.jpg" ALIGN="left" height="118" width="107"/> Natalie is a first year Ph.D. student in the Molecular Biosciences Program, co-advised with Dr. Lori Neuman-Lee. Her research integrates modeling, high-throughput sequencing, and experiments to understand disease dynamics and spread in wild vertebrates (snakes) and an emerging fungal pathogen. More about Natalie's research <a href = "https://nhaydt.owlstown.net"><font color="MediumAquaMarine">here.</font></a>  

<h3>Shelby McCormick (Undergraduate Researcher) </h3>
<p> <IMG SRC="/images/herb.jpg" ALIGN="left" height="118" width="107"/> Shelby is a junior Mathematics major. Her project focuses on training deep learning models for automated detection of phenological stage from herbarium specimen images of <i>Striga</i> spp. parasitic plants.</br>
	
<h3>Emilio Soriano Chavez (Undergraduate Researcher) </h3>
<p> <IMG SRC="/images/Emilio0SC.JPG" ALIGN="left" height="118" width="107"/> Emilio is a is a junior at A-State majoring in Computer Science. He is currently working on a project to develop new computational models to predict phenotype from environmental and genotype information in maize.</br>
	
<h3>Current opportunities:</h3>	
<p>Undergraduate students from any major are always welcome to contact Dr. Bellis and to inquire about availability of paid research opportunities. Prior coding experience is not necessary, however, you should have a deep interest in learning R and/or Python and applying computational techniques to interesting problems in biology, agriculture, or computer science! </p>

<p>We currently do not have any open graduate student positions</p>.

</article>
            
              <article id="News">
  <h1>News</h1>
  <p>
<figure class="image main">
    
        <img style="height: 80%; width: 80%; object-fit: contain" src="/images/IMG_0400.png"/>
    
    
</figure>
<b>11.22.21</b><p> Shelby's application for the Undergraduate Research Scholar was accepted. Woohooo!! Building on her work for the AI-Campus project, her research will focus on developing improved machine learning models for herbarium image analysis!

<b>4.23.21</b><p> Undergraduate student researchers Daniel and Shelby are joining our AI-Campus team to work on the herbarium image project!  Natalie Haydt is also joining the lab to pursue her PhD in MBS, co-advised with Dr. Neuman-Lee. We are so excited to welcome Daniel, Shelby, and Natalie! </p>
	
<b>4.21.21</b><p> <IMG SRC="/images/prairie.JPG" ALIGN="right" height="120" width="160"/>An article <a href='https://www.arkansasonline.com/news/2021/apr/18/tallgrass-prairie-restoration-a-priority-for-fort/'>just published</a> in the NW Arkansas Democrat Gazette about the Tallgrass Prairie Restoration Project. The article came out just as we were all together sampling hemiparasitic plant associated soils in NW AR! Here's Dr. Bellis perfecting her soil coring technique under the tutelage of collaborators Mr. Jay Randolph & Dr. Jeff Shaver! </p>
	  
<b>6.9.20</b><p> The lab is growing! Undergraduate student Emilio Soriano-Chavez joined us this summer to work on machine learning models for genomic prediction! </p>
	  
<b>10.25.19</b> Emily is off to the annual Arkansas INBRE conference, in Fayetteville, AR!

</article>

            
              <article id="contact">
  <h1>Contact</h1>
<p>You can find me in room 202 of the Arkansas Biosciences Institute on the A-State main campus in Jonesboro, or reach me by email at ebellis[at]astate[dot]edu.</p>

<ul class="icons">

	<li><a href="https://twitter.com/em_bellis" class="icon fa-twitter"><span class="label">twitter</span></a></li>

	<li><a href="https://github.com/em-bellis" class="icon fa-github"><span class="label">github</span></a></li>

</ul>

</article>

            
          </div>

        
          <div id="languages">
            
          </div>

				
					<footer id="footer">
						<p class="copyright">&copy; Untitled. Design: <a href="https://html5up.net">HTML5 UP</a>. Powered by <a href="https://www.gohugo.io/">Hugo.</a></p>
					</footer>

			</div>
		
			<div id="bg"></div>

		
			<script src="/js/skel.min.js"></script>
			<script src="/js/util.js"></script>
			<script src="/js/main.js"></script>
			

	</body>
</html>

